### PR TITLE
UI changes part 1

### PR DESCRIPTION
**Pull request by JillianK**
This PR has the final file tab design (including the file explorer), the backpack button, the clear console button, and swapping the commit code and version history buttons (all in dark mode)
Screen Shot 2021-05-25 at 2 02 18 PM

Edit: now with more compact tabs:
Screen Shot 2021-05-25 at 2 46 09 PM

## Links

- spec: [figma](https://www.figma.com/file/fZo4fcZmDUioOCaz04Gza4/Javalab-Design-Spec?node-id=0%3A1)
- spec: [slack follow up on the file explorer and tabs](https://codedotorg.slack.com/archives/C01EF4GJ9GE/p1621893673064200)
- jira ticket: [Part of UI Pass](https://codedotorg.atlassian.net/browse/CSA-361?atlOrigin=eyJpIjoiNTljZTY2ODExMTA2NDYxNTlkMjU3ZjAyMWUyNDFlMDEiLCJwIjoiaiJ9)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/javalab/JavalabEditor.jsx [hunk 1]
```diff
@@ -468,17 +474,24 @@ class JavalabEditor extends React.Component {
             isRtl={false}
             onClick={this.props.handleVersionHistory}
           />
+          <PaneButton
+            id="javalab-editor-save"
+            iconClass="fa fa-check-circle"
+            onClick={onCommitCode}
+            headerHasFocus={true}
```
Comment (maddiedierker) [inline apps/src/javalab/JavalabEditor.jsx]: what does this prop do?

also, style nit: for boolean props that are true, you can remove `={true}` (e.g., `<PaneButton headerHasFocus/>` will do the same thing)

Comment (JillianK) [reply apps/src/javalab/JavalabEditor.jsx]: I think it affects the color of the button and I can update those boolean props!

**Review by maddiedierker — APPROVED** [review]

File: apps/src/javalab/JavalabEditor.jsx [hunk 2]
```diff
@@ -442,43 +448,50 @@ class JavalabEditor extends React.Component {
...
           <PaneButton
             id="javalab-editor-create-file"
             iconClass="fa fa-plus-circle"
             onClick={() => this.setState({openDialog: CREATE_FILE})}
-            headerHasFocus={true}
+            headerHasFocus
             isRtl={false}
-            label="Create File"
-            leftJustified={true}
+            label="New File"
```
Comment (molly-moen) [inline apps/src/javalab/JavalabEditor.jsx]: should we put these button labels in i18n?

**Review by molly-moen — APPROVED** [review]
Looks good! One non-blocking question

**Pull request merged**